signer/client/signer_trust.go

```diff
@@ -180,16 +179,7 @@ func (trust *NotarySigner) ListAllKeys() map[string]string {
 // CheckHealth checks the health of one of the clients, since both clients run
 // from the same GRPC server.
 func (trust *NotarySigner) CheckHealth(timeout time.Duration) error {
-
-	// Do not bother starting checking at all if the connection is broken.
-	if trust.clientConn.State() != grpc.Idle &&
```

Comment:
thank you for removing the `State()` calls but I think we should still check the client connection status, it seems that the `grpc.ConnectivityState` is still a typed int corresponding to the same constants: https://godoc.org/google.golang.org/grpc#ConnectivityState